qutip/qip/gates.py

```diff
@@ -488,6 +509,29 @@ def sqrtiswap(N=None, targets=[0, 1]):
                               [0, 0, 0, 1]]), dims=[[2, 2], [2, 2]])
 
 
+def MS_gate(theta, N=None, targets=(0, 1)):
```

Comment:
what about being more explicit and call it `molmer_sorensen` instead of `MS_gate`? This way we also avoid using initial upper case that is used for Classes.

Reply:
`molmer_sorensen` looks good.